README.md

```diff
@@ -50,76 +46,67 @@ For the accelerator-specific features, you can install them by appending `#egg=o
 python -m pip install git+https://github.com/huggingface/optimum.git#egg=optimum[onnxruntime]
 ```
 
+## Accelerated Inference
 
-## Optimizing models towards inference
+While everything can be done in a programmatic way in 🤗 Optimum for cutomized use-cases, we also proivde a command-line interface to perform common tasks easily. 
 
-Along with supporting dedicated AI hardware for training, Optimum also provides inference optimizations towards various frameworks and
-platforms.
+### ONNX + ONNX Runtime
 
-Optimum enables the usage of popular compression techniques such as quantization and pruning by supporting [ONNX Runtime](https://onnxruntime.ai/docs/) along with Intel [Neural Compressor](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html) and OpenVINO [NNCF](https://docs.openvino.ai/latest/tmo_introduction.html).
+It is possible to export 🤗 Transformers models to the ONNX format and perform graph optimization as well as quantization easily:
 
-| Features                           | ONNX Runtime          |     Neural Compressor   |         OpenVINO        |
-|:----------------------------------:|:---------------------:|:-----------------------:|:-----------------------:|
-| Post-training Dynamic Quantization |  :heavy_check_mark:   |    :heavy_check_mark:   |          N/A            |
-| Post-training Static Quantization  |  :heavy_check_mark:   |    :heavy_check_mark:   |    :heavy_check_mark:   |
-| Quantization Aware Training (QAT)  |  Stay tuned! :star:   |    :heavy_check_mark:   |    :heavy_check_mark:   |
-| Pruning                            |        N/A            |    :heavy_check_mark:   |    :heavy_check_mark:   |
```

Comment:
I do not think this is needed here, it should be more in the documentation. 
Or otherwise I think we should be showing instead of telling via a small command example.

Reply:
We cannot add an example for each, it would make the README overcrowded and unclear. The README is the perfect place to present the different features of each subpackages, in order for users to be redirected towards what they need. We also should have it in the documentation.

Reply:
Done